operators-manual/companion-web.md

```diff
@@ -10,19 +16,28 @@ The web interface by default opens at the [Network](http://192.168.2.2:2770/netw
 - Check WiFi and Internet Status 
 - Configure Ethernet IP address
 - Test the network for Uploads and Downloads
+- Configure Companion network mode: In advanced mode, the [Network](http://192.168.2.2:2770/network) page features advanced network configuration options for the Companion computer raspberry pi. Companion can be configured as DHCP Server, DHCP Client or Manual mode. The advanced mode can be accessed with the Advanced Options toggle switch on the web interface header.
+  - Manual: Companion has a static IP(192.168.2.2) in this mode. This is the default mode that is pre-configured on a fresh Companion Image.
```

Comment:
Manual: The interface is configured with a static IP address. This is the default configuration. The default IP address is 192.168.2.2.